docs/root/install/tools/route_table_check_tool.rst

```diff
@@ -8,10 +8,10 @@ The tool can also be used to check whether a path redirect, path rewrite, or hos
 match what is expected.
 
 Input
-  The tool expects two input JSON files:
```

Comment:
Is there any particular reason we can't support a v2 config in JSON format like we do for main server configs? Some people prefer JSON over YAML and it seems like we could support both?

Reply:
I think it already will support both based on my understanding of the YAML parser. I guess we can explicitly test and call that out in the docs.

Reply:
Just verified, v2 jsons also work. I'll update the docs.